core/models.py

```diff
@@ -7,7 +7,7 @@ class Person(models.Model):
     date_of_birth = models.DateField(blank=True, null=True)
     address = models.CharField(max_length=256, blank=True, null=True)
     age = models.IntegerField(blank=True, null=True)
-    nhs_number = models.IntegerField(blank=True, null=True)
+    nhs_number = models.CharField(max_length=70, blank=True, null=True)
```

Comment:
@tab1tha I don't think we need a length of 70 as the NHS is a 10 digit number by definition.  Change that and I'll approve.